src/storage/LocalStorage.js

```diff
@@ -0,0 +1,11 @@
+export const getResume = storage => {
+  return JSON.parse(storage.getItem('resume'));
+};
+
+export const saveResume = (storage, resume) => {
+  storage.setItem('resume', JSON.stringify(resume));
```

Comment:
make it a constant and use it everywhere mentioned

Reply:
done